website/static/install.sh

```diff
@@ -21,11 +22,12 @@ help() {
     # Display Help
     echo "Install script for Oh My Posh"
     echo
-    echo "Syntax: install.sh [-h|d|t]"
+    echo "Syntax: install.sh [-h] [-d <directory>] [-t <directory>] [-v <version>]"
     echo "options:"
     echo "-h     Print this help."
     echo "-d     Specify the installation directory. Defaults to $HOME/bin, $HOME/.local/bin or the directory where oh-my-posh is installed."
     echo "-t     Specify the themes installation directory. Defaults to the oh-my-posh cache directory."
+    echo "-v     Version to download"
     echo
 }
 
```

Comment:
To include the `-v` option for specifying a version, you should modify the `getopts` string. Here's the change:

```
while getopts ":hd:t:v:" option; do
```

This will allow the script to handle the `-v` option correctly, requiring an argument for the version.